post_1.md

```diff
@@ -1,30 +1,34 @@
 # Getting the Fuck out of Teaching
 
 ## The world of teaching
-
 As a young idealistic student planning my future career I told myself I could never work in a corporate environment. How could I give up my time to line someone else's pockets? Should I not being doing something to give back to the world? It was with this in mind I decided to become a maths teacher. I had some experience tutoring people and I had always dreamed of being a teacher, as it turns out achieving your dreams isn't all it's made out to be sometimes.

```

Comment:
s/young/young,/
s/career/career,/
s/being\ doing/be doing/
s/in\ mind\ I/in mind that I/    --(not sure about this one)
s/teacher,\ as/teacher. As/
s/out\ achieving/out, acheiving/